avm/res/api-management/service/product/main.bicep

```
@@ -101,6 +104,19 @@ module product_groups 'group/main.bicep' = [
   }
 ]
 
+module product_policies 'policy/main.bicep' = [
+  for (policy, index) in policies ?? []: {
+    name: '${deployment().name}-Policy-${index}'
+    params: {
+      apiManagementServiceName: apiManagementServiceName
+      productName: name
+      name: policy.?name ?? 'policy'
+      format: policy.?format ?? 'xml'
```

```suggestion
      name: policy.?name
      format: policy.?format
```
Will automatically pull the parameter defaults